core/iwasm/common/wasm_application.c

```diff
@@ -107,7 +107,22 @@ execute_main(WASMModuleInstanceCommon *module_inst, int32 argc, char *argv[])
        the actual main function. Directly calling main function
        may cause exception thrown. */
     if ((func = wasm_runtime_lookup_wasi_start_function(module_inst))) {
-        return wasm_runtime_call_wasm(exec_env, func, 0, NULL);
+        const char *wasi_proc_exit_exception = "wasi proc exit";
```

Comment:
Is it better if we use a global variable or macro to represent the string "wasi proc exit"？ It is a very import constant string now。

Reply:
ideally we should stop using strings i guess.